app/services/db/monthly_sp_auth_count/unique_monthly_auth_counts_by_partner.rb

```diff
@@ -93,23 +93,51 @@ def call(key:, issuers:, start_date:, end_date:)
       # @return [Array<String>]
       def build_queries(issuers:, months:)
         months.map do |month_range|
+          today = Date.today
           params = {
             range_start: month_range.begin,
             range_end: month_range.end,
             year_month: month_range.begin.strftime('%Y%m'),
             issuers: issuers,
+            one_years_ago: today - 365,
+            two_years_ago: today - 2 * 365,
+            three_years_ago: today - 3 * 365,
+            four_years_ago: today - 4 * 365,
+            five_years_ago: today - 5 * 365,
           }.transform_values { |value| quote(value) }
 
+
+
           format(<<~SQL, params)
             SELECT
               sp_return_logs.user_id
             , %{year_month} AS year_month
             , COUNT(sp_return_logs.id) AS auth_count
             , sp_return_logs.ial
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{now) AND %{one_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year1
```

Comment:
I think we want to be looking at `MIN(sp_return_logs.returned_at) BETWEEN %{range_end} and  %{one_year_ago}`

From the doc
> Age of IDV will be calculated as of the age of the proofed data at **the first time of login within that month** (ex proofing data from April 2, 2023 would be considered less than one year old if the user signs back in on April 1, 2024 for the first time).

- I think that `MIN(sp_return_logs.returned_at)` is the first time of that month
- I think that `now` is vague and we want reports we can re-run consistently later, so I think `range_end` (aka `month_range.end`) is the right value
- `one_year_ago` (singular) I think reads better

Reply:
This has been addressed